### PR TITLE
[IMP] survey: move print in action menu

### PR DESCRIPTION
**Pull request by amdi-odoo**
Move the print button of the survey form view into the action menu.

Task-3819958

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by Odoonan — COMMENTED** [review]
Hello @amdi-odoo ,

Just small :socks: remarks, LGTM otherwise :)

Cheers

File: addons/survey/views/survey_survey_views.xml [hunk 1]
```diff
@@ -195,6 +194,19 @@
             </form>
         </field>
     </record>
+
+    <record id="action_print_survey" model="ir.actions.server">
```
Comment (awa-odoo) [inline addons/survey/views/survey_survey_views.xml:198]: Can we make a separate file "ir_actions_server_data.xml" for this? Seems to be the most used convention.

**Review by awa-odoo — COMMENTED** [review]
Hello @amdi-odoo 

A couple light comments on this one.

Thanks & cheers!

**Comment by awa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**